packages/labs/react/src/create-component.ts

```diff
@@ -90,8 +90,22 @@ type EventListeners<R extends EventNames> = {
     : (e: Event) => void;
 };
 
+// Interface that includes parts of the React module that we use. This allows
+// users to construct an object with just the needed methods and prevents type
+// incompatibilities when there are multiple React types being used.
+interface ReactLike {
```

Comment:
Should we export this? I could see someone using this with `satisfies` to make an object to pass to several createComponent calls. If so, make the comment a doc comment.